lang/java/ipc-netty/src/test/java/org/apache/avro/ipc/netty/TestNettyServerWithCompression.java

```diff
@@ -31,7 +31,7 @@ public class TestNettyServerWithCompression extends TestNettyServer {
   public static void initializeConnections() throws Exception {
     initializeConnections(ch -> {
       ch.pipeline().addFirst("deflater", new JdkZlibEncoder(6));
-      ch.pipeline().addFirst("inflater", new JdkZlibDecoder());
+      ch.pipeline().addFirst("inflater", new JdkZlibDecoder(0));
```

Comment:
The 0-argument constructor has been deprecated in favour of this one.

The parameter value is the same as with the deprecated constructor.